terraform/environments/electronic-monitoring-data/glue_data.tf

```diff
@@ -57,7 +57,61 @@ data "aws_iam_policy_document" "dms_dv_s3_iam_policy_document" {
       aws_s3_bucket.dms_dv_parquet_s3_bucket.arn,
       "${aws_s3_bucket.dms_dv_parquet_s3_bucket.arn}/*",
       aws_s3_bucket.dms_dv_glue_job_s3_bucket.arn,
-      "${aws_s3_bucket.dms_dv_parquet_s3_bucket.arn}/*",
+      "${aws_s3_bucket.dms_dv_glue_job_s3_bucket.arn}/*",
+    ]
+  }
+}
+
+data "aws_iam_policy_document" "dms_dv_athena_iam_policy_document" {
+  statement {
+    effect = "Allow"
+    actions = [
+      "athena:TerminateSession",
+      "athena:CreateDataCatalog",
+      "athena:UpdateDataCatalog",
+      "athena:GetTableMetadata",
+      "athena:StartQueryExecution",
+      "athena:GetSession",
+      "athena:DeleteWorkGroup",
+      "athena:GetQueryResults",
+      "athena:GetDatabase",
+      "athena:GetDataCatalog",
+      "athena:UpdateWorkGroup",
+      "athena:CreateWorkGroup",
```

Comment:
if youre not creating a custom workgroup for this work, do you need this permission?